pkg/local_object_storage/blobstor/compress.go

```diff
@@ -0,0 +1,35 @@
+package blobstor
+
+import (
+	"github.com/klauspost/compress/zstd"
+)
+
+func noOpCompressor(data []byte) []byte {
+	return data
+}
+
+func noOpDecompressor(data []byte) ([]byte, error) {
+	return data, nil
+}
+
+func zstdCompressor() func([]byte) []byte {
+	enc, err := zstd.NewWriter(nil)
+	if err != nil {
+		panic(err)
+	}
+
+	return func(data []byte) []byte {
+		return enc.EncodeAll(data, make([]byte, 0, len(data)))
+	}
+}
+
+func zstdDecompressor() func([]byte) ([]byte, error) {
+	dec, err := zstd.NewReader(nil)
+	if err != nil {
+		panic(err)
```

Comment:
`zstd.NewReader(nil)`  call never returns an error. We can:

1. ignore error;

2. panic on error (like in code);

3. return error. In this case we need to refactor `Option` to return error, and constructor too.

I don't like 3 since we don't know what to do with this error, and application will eventually shutdown anyway.

What do u think?

Reply:
Is there a way to leave the data uncompressed and log a error?

Reply:
@realloc yes

Reply:
Then let's do 4) + log error.

Reply:
Rdy.